src/constants.ts

```diff
@@ -4,3 +4,5 @@ export enum Capabilities {
   AuxiliaryFunds = 'auxiliaryFunds',
   PaymasterService = 'paymasterService',
 }
+export const DEFAULT_PRIVACY_URL = 'https://base.org/privacy-policy';
+export const DEFAULT_TERMS_URL = 'https://base.org/terms-of-service';
```

Comment:
I don't think these are the right privacy and terms of service to be using as defaults?

Reply:
Will double check with Mind